trio/_tests/verify_types.json

```diff
@@ -7,11 +7,11 @@
     "warningCount": 0
   },
   "typeCompleteness": {
-    "completenessScore": 0.9186602870813397,
+    "completenessScore": 0.9202551834130781,
     "exportedSymbolCounts": {
-      "withAmbiguousType": 0,
-      "withKnownType": 576,
-      "withUnknownType": 51
+      "withAmbiguousType": 1,
```

Comment:
I can't seem to figure out how to run check_type_completeness properly. It says it can't find module trio to check.

Reply:
adding a `py.typed` file and installing trio without the `-e` flag should do it (don't ask me why though). Running `pyright --verifytypes` probably fails the same way. Otherwise I can share my tox config for it tomorrow

Reply:
Here's the error:
```
trio._ssl.SSLStream.transport_stream
  f:\Git\clones\python\trio\trio\_ssl.py:764:18 - error: Type is missing type annotation and could be inferred differently by type checkers
    Inferred type is "Stream"
  f:\Git\clones\python\trio\trio\_ssl.py:764:18 - error: Type is missing type annotation and could be inferred differently by type checkers
    Inferred type is "None"
```
Specifically, this line:
```py
self.transport_stream = None  # type: ignore[assignment]  # State is CLOSED now, nothing should use
```
You'll want to add a variable annotation for that, since it's just `Stream` in `__init__`.

Reply:
Have you read through point 3 in my first comment for discussing this PR? The docstring says it's required to be a `Stream` object and nothing anywhere else is checking to make sure `transport_stream` is valid.

Reply:
It might be confusing Pyright still, since it's using both assignment statements to infer the type. If you add an annotation to the one in `__init__()`, it should fix the error. Despite the name, `type: ignore` just disables errors, type checkers still have to analyse that line of code to figure out what it does...

Reply:
Fixed in c896899